assets/app/view/game/par.rb

```diff
@@ -33,8 +33,10 @@ def render
             },
             on: { click: par },
           }
-          purchasable_shares = (entity.cash / share_price.price).to_i
-          text = "#{@game.format_currency(share_price.price)} (#{purchasable_shares})"
+          purchasable_shares = [(entity.cash / share_price.price).to_i, @corporation.max_ownership_percent / 10].min
```

Comment:
shares aren't always 10, i think you want total_shares

Reply:
Thanks, that makes sense. I poked around in 1817 to see if that case might come up, but didn't see anywhere that it did. I'll get it changed.